tests/test_mpi.py

```diff
@@ -2727,6 +2801,85 @@ def test_adjoint_F_no_omp(self, mode):
         self.run_adjoint_F(3)
 
 
+class TestElastic:
```

Comment:
This is stronger than the mfe, as it also looks for correct grouping of functions in a halospot.
The MFE does not need to merge functions

Reply:
No similar test exists in the codebase